paddle/fluid/operators/math/blas.h

```diff
@@ -125,7 +125,8 @@ class Blas {
                       const MatDescriptor& dim_a,
                       const framework::Tensor& mat_b,
                       const MatDescriptor& dim_b, T alpha, int head_number,
-                      framework::Tensor* mat_out, T beta) const;
+                      bool mat_y_split_vertical, framework::Tensor* mat_out,
```

Comment:
why not extend mat_y_split_vertical at the end ? and make mat_y_split_vertical=false as default.

Reply:
Will add mat_y_split_vertical at the end to keep it consistent with defination of BatchedGEMMWithHead. But to make it false as default lead to compiling error. I googled it and seems it is not easy to fix it.
https://stackoverflow.com/questions/4050202/why-default-argument-cannot-be-specified-for-an-explicit-template-specialization
since in our case, both true and false were equally been used. so I keep that parameter as non default.